src/einsteinpy/symbolic/predefined/find.py

```diff
@@ -0,0 +1,25 @@
+import importlib
+
+
+def find(search_string):
+    """
+    Performs a find operation on available functions.
+
+    Parameters
+    ----------
+        search_string : str
+            Name of the function to be searched.
+
+    Returns
+    -------
+    list
+        A list of available functions related to search_string.
+
+    """
+    current_module = "einsteinpy.symbolic.predefined"
```

Comment:
Add another line before this.
`search_string = search_string.lower()`.
Then it can be always ensured that search happens in lowercase, whatever the user inputs.